### PR TITLE
[SP-4812] MetaApp native message support

### PR DESCRIPTION
**Pull request by Suryakantj**

### CODE REVIEW

File: metaapp/src/androidTest/java/com/sourcepointmeta/metaapp/MetaAppTests.java [hunk 1]
```diff
@@ -151,13 +159,14 @@ public void checkNoMessageDisplayWithShowMessageOnce() {
...
         navigateBackToListView();
+        Assert.assertTrue(checkForPropertyListScrren());
         tapOnProperty();
         Assert.assertFalse(checkWebViewDisplayedFor(MESSAGE));
         Assert.assertTrue(checkFor(CONSENTS_ARE_DISPLAYED));
     }
 
     @Test
-    public void resetConsentDataAndCheckForMessageWithShowMessageOnce() {
+    public void resetConsentDataAndCheckForMessageWithShowMessageOnce() throws InterruptedException {
```
Comment (andresilveirah) [inline metaapp/src/androidTest/java/com/sourcepointmeta/metaapp/MetaAppTests.java:169]: @Suryakantj why do we have to declare `throws InterruptedException`?

File: metaapp/src/main/java/com/sourcepointmeta/metaapp/adapters/PropertyListAdapter.java [hunk 2]
```diff
@@ -71,6 +72,7 @@ public void onBindViewHolder(@NonNull PropertyViewHolder viewHolder, int positio
         viewHolder.propertyNameTextView.setText(name);
         viewHolder.propertyAccountID.setText("Account ID : " + accountID);
         viewHolder.propertyCampaign.setText("Campaign : " + campaign);
+        viewHolder.propertyType.setText("PropertyType : "+ propertyType);
```
Comment (andresilveirah) [inline metaapp/src/main/java/com/sourcepointmeta/metaapp/adapters/PropertyListAdapter.java:75]: I believe we could rename `propertyType` to `messageType`. The property has no type, it's the message that can be web or native.

File: metaapp/src/androidTest/java/com/sourcepointmeta/metaapp/MetaAppTests.java (hunk 1 above)
Comment (Suryakantj) [reply metaapp/src/androidTest/java/com/sourcepointmeta/metaapp/MetaAppTests.java:169]: @andresilveirah We are using wait period till message is visible , for this we are using "countdown" latch. which can throw InterruptedException.

File: metaapp/src/main/java/com/sourcepointmeta/metaapp/adapters/PropertyListAdapter.java (hunk 2 above)
Comment (Suryakantj) [reply metaapp/src/main/java/com/sourcepointmeta/metaapp/adapters/PropertyListAdapter.java:75]: @andresilveirah Yes we can rename it , I would change it to messageType.

**Pull request merged**